ATM/ATM.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+namespace ATM
+{
+    public static class ATM
+    {
+        public static BankAccount bankAccount = new BankAccount();
+        public static bool exit = false;
+
+        public static void ShowMenu()
+        {
+            Console.WriteLine("Welcome to DVCCU!");
+            Console.WriteLine("Please select an option:");
+            while(!exit)
+            {
+                Console.WriteLine("1: View your Balance");
+                Console.WriteLine("2: Make a withdraw");
+                Console.WriteLine("3: Deposit funds");
+                Console.WriteLine("4: Exit");
+
+                string userSelection = Console.ReadLine();
+
+                if (userSelection == "1")
+                {
+                    bankAccount.ViewBalance();
+                }
+                else if (userSelection == "2")
+                {
+                    PromptForWithdraw();
+                }
+                else if (userSelection == "3")
+                {
+                    PromptForDeposit();
+                }
+                else if (userSelection == "4")
+                {
+                    Console.WriteLine("Thank you for banking with DVCCU!");
+                    exit = true;
+                }
+                else
+                {
+                    Console.WriteLine("To choose a valid option, please enter the number for the type of transaction you would like to perform.");
+                }
+            }
+            
+        }
+
+        public static void PromptForWithdraw()
+        {
+            Console.WriteLine("How much would you like to withdraw from your account?");
+            string userInput = Console.ReadLine();
+
+            try { 
+                int intToWithdraw = Convert.ToInt32(userInput);
+                decimal amountToWithdraw = Convert.ToDecimal(intToWithdraw);
```

Comment:
Converting to `int` first will drop your decimals. Not sure if that was intentional?
```suggestion
                decimal amountToWithdraw = Convert.ToDecimal(userInput);
```

Reply:
It was done under the impression that converting from string to decimal wouldn't work without converting to int first.

Reply:
> It was done under the impression that converting from string to decimal wouldn't work without converting to int first.

It absolutely works (for valid-looking decimals, of course).